scripts/prepare_navit_macos.sh

```diff
@@ -0,0 +1,19 @@
+# Use this script to build navit on a Mac
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+brew install glib gtk+ gettext libpng protobuf-c cmake librsvg imagemagick gpsd
+brew services start gpsd
+
+#TODO: this needs to be adapted for the PR: git clone https://github.com/navit/navit.git
+git clone https://github.com/OLFDB/navit.git
+cd navit
+
+#TODO: this needs to be adapted for the PR: remove
+git checkout macosbuild
+git fetch
+#remove end
+
+mkdir build
+cd build
+cmake ../contrib/macos/
+make install
+open /usr/local/bin/navit.app
```

Comment:
Also not good for the ci

Reply:
For CI I think macos is out of scope, as the macos CI is only available in a paid account. Using macos SDK to crosscompile is violating the Apple license so this can't be used as well.

Reply:
Not on [github actions](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on) where i want to move to soon

Reply:
Great. I was not aware of that. Maybe we can find a way to use my developer certificate to sign the build then (I could provide an ssh access to upload the app package to my machine, hava a service running checking for a new upload, sign it and somehow trigger CI to download the result again) and even bring navit to the app store. Don't know if any license restrictions will prohibit that. I guess there was some discussion in one of the issue conversations about that.

Reply:
I think there where problems with having a gpl license...